File: 1618.py
# Introductory Problems : Bit Strings
#  https://cses.fi/problemset/task/1617/

# GIF of powers of 5

n=int(input())
i, c = 5, 0
while n/i >= 1:
    c+=int(n/i)
    i*=5
print(int(c))

File: 1618.cc
// Introductory Problems : Trailing Zeros
// https://cses.fi/problemset/task/1618/

#include <iostream>
using namespace std;

int main() {
	int n;
	cin >> n;
	int c = 0;
	for (int i=5;n/i>=1;i*=5)
			c+=n/i;
	cout << c;
}
